README.md

```diff
@@ -71,21 +71,25 @@ After installing, you can follow this simple steps:
 $ git clone git@github.com:ProyectoIntegrador2018/makers.git
 ```
 
-2. Navigate to the `makers/` directory and run:
+2. Add Needed Ruby Enviormental variables
+
+They should be added into a the file 'config/local_env.yml' or be setup into your enviorment
```

Comment:
remove "the" before file

Reply:
Also, it might be wise to mention that the variables that need to be added are `TEC_USERNAME` and `TEC_PASSWORD`, so that future devs don't have to dig into the code to see what variables need be set.

Reply:
And/or maybe add a sample `local_env.yaml` file?